ansible/roles/stack-sunbird/templates/transformer.env

```diff
@@ -72,6 +72,14 @@ POSTHOG_EVENT_ENABLED=FALSE
 
 #Azure Config
 AZURE_BLOB_STORE_CONTAINER={{sunbird_azure_uci_container_name | default('uci-' + env )}}
-AZURE_BLOB_STORE_ACCOUNT_KEY={{sunbird_private_storage_account_name}}
-AZURE_BLOB_STORE_ACCOUNT_NAME={{sunbird_private_storage_account_key}}
-SELECTED_FILE_CDN=azure
\ No newline at end of file
+AZURE_BLOB_STORE_ACCOUNT_NAME={{sunbird_private_storage_account_name}}
```

Comment:
@surabhi-mahawar Are these variable still required? Below is another variable which has same contents
`SUNBIRD_CLOUD_MEDIA_STORAGE_KEY`

Reply:
No this is not required not, I have removed these variables from the files